### PR TITLE
style: Enhanced UI of searchView and added animation

### LINKED ISSUE #2431: Searchbar UI enhancement
**Current UI of searchbar isn't very nice, would like to add animation when search icon is clicked**

**Screenshots**

untitled

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by pallav12**
Fixes #2431

Changes: Fixed color, background and added animation on search icon clicked.

Screenshots for the change: 
screenRecordAtm

### CODE REVIEW

**Comment by atm1504** [comment]
@pallav12 please provide a better gif

**Comment by pallav12** [comment]
Updated the gif

File: app/src/main/res/layout/activity_chat.xml [hunk 1]
```diff
@@ -4,37 +4,42 @@
...
+            android:imeOptions="actionSearch"
             android:paddingLeft="@dimen/padding_moderate"
-            android:background="@color/colorPrimaryDark"
-            android:textSize="@dimen/text_size_normal"
             android:singleLine="true"
-            android:imeOptions="actionSearch"/>
+            android:textSize="@dimen/text_size_normal"
+            android:visibility="gone" />
+
+        <FrameLayout
```
Comment (atm1504) [inline app/src/main/res/layout/activity_chat.xml:29]: why?

Comment (pallav12) [reply app/src/main/res/layout/activity_chat.xml:29]: FrameLyout is added for fine blackish link below EditText to create a separation from RecyclerView

**Review by atm1504 — APPROVED** [review]

**Pull request merged**